api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
I'm pretty sure they'll all have a signature of some kind so you might as well just add the space to the printf.